client/src/translation/en.json

```diff
@@ -30,7 +30,7 @@
     "about": "About",
     "intro": "Mirlo provides a user-friendly space to help musicians sell music, manage subscriptions, and share with their supporters.",
     "ourMission": "Our mission",
-    "missionStatement": "<1>The music industry does not work for musicians or listeners and needs a radical re-imagination. </1><1>Mirlo is a community of musicians, listeners, and coders who are daring to do just that: taking lessons learned in working in the solidarity economy and applying them to our process and product.</1><1>We are building an online audio distribution (think Bandcamp) and patronage (think Patreon) platform that aims to be radical, accessible, open source (free & libre), modular, and standards based.</1>",
+    "missionStatement": "<1>The music industry does not work for musicians or listeners and needs a radical re-imagination. </1><1>Mirlo is a community of musicians, listeners, and coders who are daring to do just that, taking lessons learned in working in the solidarity economy and applying them to our process and product.</1><1>We are building an online audio distribution (think Bandcamp) and patronage (think Patreon) platform that aims to be radical, accessible, open source (free & libre), modular, and standards based.</1>",
```

Comment:
colon is correct here I think?

`We're doing just that: things we're doing`

Reply:
I think the sentence is instead trying to imply:
`We're doing just that, how we're doing it`
It's the next sentence that explains what we're doing.